app/Http/Controllers/QuestionApiController.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Question;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class QuestionApiController extends Controller
+{
+    public function create(Request $request): JsonResponse
+    {
+        $newQuestion = new Question;
+        $request->validate([
+            'question' => 'string',
+            'hint' => 'nullable|string',
+            'points' => 'integer',
+            'quiz_id' => 'integer',
+        ]);
```

Comment:
All of the fields other than hint should be set as required.

You should also add the following:
- Max lengths, need to make sure the question and hint provided aren't too long to go into the database
- An exists check on the quiz_id, make sure it actually exists in the quizzes table

Annnnnnd then, you should look at moving this validation into a Request object to make it reusable, and to tidy up this controller.